content/hi/application-programming-interface.md

```diff
@@ -0,0 +1,18 @@
+---
+title: अनुप्रयोग प्रोग्रामिंग का अंतरापृष्ठ (Application Programming Interface)
+status: Feedback Appreciated
+category: संकल्पना
+tags:  ["संरचना","मूलभूत"]
+---
+
+## यह क्या है
+
+एक एपीआई कंप्यूटर प्रोग्राम के लिए संवाद स्थापित करने का एक माध्यम है। जिस तरह मनुष्य एक वेब पेज के माध्यम से एक वेबसाइट के साथ सूचना का आदान-प्रदान करते हैं, एक एपीआई विभिन्न कंप्यूटर प्रोग्राम को एक दूसरे को सूचना का आदान-प्रदान करने की अनुमति देता है।  मानवीय अंतःक्रियाओं के विपरीत, एपीआई की सीमाएँ होती हैं कि उनसे क्या पूछा जा सकता है और क्या नहीं। सहभागिता के बीच सीमा स्थापित करने से अलग-अलग प्रोग्राम के बीच स्थिर और कार्यात्मक संचार बनाने में मदद मिलती है।
+
+## समस्या जिस का यह समाधान करता है
+
+जैसे-जैसे एप्लिकेशन अधिक जटिल होते जाते हैं, कोड में छोटे परिवर्तन भी दूसरी कार्यक्षमताओं पर भारी प्रभाव डाल सकते हैं। अनुप्रयोगों को उनकी कार्यक्षमता के लिए एक प्रतिरुपक दृष्टिकोण रखने की आवश्यकता होती है, यदि वे एक साथ विकास और स्थिरता बनाए रख सकते हैं।  एपीआई के बिना, अनुप्रयोगों के बीच बातचीत के लिए एक रूपरेखा का अभाव रहता है। एक साझा ढांचे के बिना, अनुप्रयोगों को [स्केल](/scalability/) और एकीकृत करना चुनौतीपूर्ण है।
+
+## यह कैसे मदद करता है
+
+एपीआई कंप्यूटर प्रोग्राम या एप्लिकेशन को एक निर्धारित और समझने योग्य तरीके से जानकारी साझा करने और साझा करने की अनुमति देता है। ये आधुनिक अनुप्रयोगों के लिए एक मूलभूत अंग हैं और ये डेवलपर्स को अनुप्रयोगों को एक साथ एकीकृत करने का एक तरीका प्रदान करते हैं। जब भी आप एक साथ काम करने वाली [माइक्रोसर्विसेज]((/microservices/)) के बारे में सुनते हैं, तो आप अनुमान लगा सकते हैं कि वे एक एपीआई के माध्यम से संवाद करती हैं।
```

Comment:
```suggestion
एपीआई कंप्यूटर प्रोग्राम या एप्लिकेशन को एक निर्धारित और समझने योग्य तरीके से जानकारी साझा करने और साझा करने की अनुमति देता है। ये आधुनिक एप्लीकेशन के लिए एक मूलभूत अंग हैं और ये डेवलपर्स को एप्लीकेशन को एक साथ एकीकृत करने का एक तरीका प्रदान करते हैं। जब भी आप एक साथ काम करने वाली [माइक्रोसर्विसेज]((/microservices/)) के बारे में सुनते हैं, तो आप अनुमान लगा सकते हैं कि वे एक एपीआई के माध्यम से संवाद करती हैं।
```

Reply:
Hi @anubha-v-ardhan! I've incorporated these changes in the latest commits. Please let me know if there's anything that can be improved. Thanks.